packages/theme/style/index.css

```diff
@@ -13,3 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.lm-DockPanel {
+  background: var(--jp-layout-color0);
```

Comment:
what exactly is this css for? I'm unsure if we should be making any blanket changes to lumino css classes. Does the targeted class have an elyra specific class we could target instead?

Reply:
i believe there may be a dark mode issue with JupyterLab (which i was going to open an issue in JupyterLab). since it it a simple enough, i included here in the meantime. this change removes the white background in the main dock panel when in dark mode:

before:

![image](https://user-images.githubusercontent.com/13156555/86660431-baf92800-bfb8-11ea-871e-7da518497dd5.png)

after:

![image](https://user-images.githubusercontent.com/13156555/86660454-bfbddc00-bfb8-11ea-8a1c-b82eae64e1c4.png)

Reply:
ooooo, pretty, I like it. and you should definitely open a PR fixing this in Lab